app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -26,11 +26,14 @@ def perform(_now)
     InPersonEnrollment.needs_usps_status_check(...5.minutes.ago).each do |enrollment|
       # Record and commit attempt to check enrollment status to database
       enrollment.update(status_check_attempted_at: Time.zone.now)
-      unique_id = enrollment.usps_unique_id
+
+      enrollment.update(unique_id: enrollment.usps_unique_id) if enrollment.unique_id.blank?
```

Comment:
Can we add a ticket to come back and remove this code?

Reply:
Follow up ticket created: https://cm-jira.usa.gov/browse/LG-7024